core/tls-websocket-server/src/common.rs

```diff
@@ -0,0 +1,87 @@
+/*
+	Copyright 2021 Integritee AG and Supercomputing Systems AG
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+
+#[cfg(all(not(feature = "std"), feature = "sgx"))]
+use crate::sgx_reexport_prelude::*;
+
+use crate::{WebSocketError, WebSocketResult};
+use rustls::NoClientAuth;
+use std::{format, io::BufReader, string::ToString, sync::Arc, vec, vec::Vec};
+
+#[cfg(feature = "std")]
+use std::fs;
+
+#[cfg(all(not(feature = "std"), feature = "sgx"))]
+use std::untrusted::fs;
+
```

Comment:
Utility functions to get the TLS configuration for the server, based on certificate and private key files. This is essentially copied from example code.